google/cloud/sql/integration_tests/CMakeLists.txt

```diff
@@ -53,3 +53,5 @@ function (google_cloud_cpp_sql_define_integration_tests)
         return()
     endif ()
 endfunction ()
+
+google_cloud_cpp_sql_define_integration_tests()
```

Comment:
Does this need to be guarded by an `if (BUILD_TESTING)` conditional?  (All similar ones seem to be.)

Reply:
> Does this need to be guarded

it is: https://github.com/googleapis/google-cloud-cpp/blob/8b4fc2f96604b704b2629dc7a5e454d34af4e533/google/cloud/sql/CMakeLists.txt#L126

---

We don't need to define a `function`, then call it. We can just run the things inside.

Reply:
OK, thanks.  I'll just note that sometimes we have `add_subdirectory(integration_tests)` being conditional, and sometimes not.